modules/cluster/src/main/java/io/fluo/cluster/runner/ScanOptions.java

```diff
@@ -40,6 +40,10 @@
   @Parameter(names = {"-h", "-help", "--help"}, help = true, description = "Prints help")
   public boolean help;
 
+  @Parameter(names = {"-esc", "--escape-non-ascii"}, help = true,
+      description = "Hex encode non ascii bytes")
+  public boolean hexEncNonAscii = false;
```

Comment:
It probably should be true by default, spewing binary data to the terminal is a mess.   I was wondering how to set it to false and found the following.  

https://github.com/cbeust/jcommander/issues/129